lib/dry/schema/messages/abstract.rb

```diff
@@ -107,21 +122,6 @@ def looked_up_paths(predicate, options)
           filled_lookup_paths(tokens)
         end
 
-        # Try to find a message for the given predicate and its options
-        #
-        # @api private
-        #
-        # rubocop:disable Metrics/AbcSize
-        def lookup(predicate, tokens, options)
```

Comment:
I removed this because it was marked as `@private` and it seemed simpler to combine this into `[]`.